electri/electri.ass

```diff
@@ -511,83 +511,83 @@ Dialogue: 0,0:09:56.13,0:09:58.04,Chinese,,0,0,0,,
 Dialogue: 0,0:09:58.04,0:10:01.42,English,,0,0,0,,that the energy flux is going in one direction.
 Dialogue: 0,0:09:58.04,0:10:01.42,Chinese,,0,0,0,,
 Dialogue: 0,0:10:02.26,0:10:05.54,English,,0,0,0,,You might think this is just an academic discussion
-Dialogue: 0,0:10:02.26,0:10:05.54,Chinese,,0,0,0,,
+Dialogue: 0,0:10:02.26,0:10:05.54,Chinese,,0,0,0,,你或许觉得  这只是一次学术讨论
 Dialogue: 0,0:10:05.54,0:10:07.89,English,,0,0,0,,that you could see the energy as transmitted
-Dialogue: 0,0:10:05.54,0:10:07.89,Chinese,,0,0,0,,
+Dialogue: 0,0:10:05.54,0:10:07.89,Chinese,,0,0,0,,并且认为  它的传输介质
 Dialogue: 0,0:10:07.89,0:10:11.40,English,,0,0,0,,either by fields or by the current in the wire.
-Dialogue: 0,0:10:07.89,0:10:11.40,Chinese,,0,0,0,,
+Dialogue: 0,0:10:07.89,0:10:11.40,Chinese,,0,0,0,,既可以是场  也可以是电线中的电流
 Dialogue: 0,0:10:11.40,0:10:13.58,English,,0,0,0,,But that is not the case,
-Dialogue: 0,0:10:11.40,0:10:13.58,Chinese,,0,0,0,,
+Dialogue: 0,0:10:11.40,0:10:13.58,Chinese,,0,0,0,,但事实并非如此
 Dialogue: 0,0:10:13.58,0:10:15.26,English,,0,0,0,,and people learned this the hard way
-Dialogue: 0,0:10:13.58,0:10:15.26,Chinese,,0,0,0,,
+Dialogue: 0,0:10:13.58,0:10:15.26,Chinese,,0,0,0,,人们是以艰难的方式明白这一点的
 Dialogue: 0,0:10:15.26,0:10:18.61,English,,0,0,0,,when they started laying undersea telegraph cables.
-Dialogue: 0,0:10:15.26,0:10:18.61,Chinese,,0,0,0,,
+Dialogue: 0,0:10:15.26,0:10:18.61,Chinese,,0,0,0,,那是在他们开始铺设海底电报电缆的时候
 Dialogue: 0,0:10:18.61,0:10:22.71,English,,0,0,0,,The first Trans Atlantic cable was laid in 1858.
-Dialogue: 0,0:10:18.61,0:10:22.71,Chinese,,0,0,0,,
+Dialogue: 0,0:10:18.61,0:10:22.71,Chinese,,0,0,0,,第一条跨大西洋海底电缆在1858年铺设
 Dialogue: 0,0:10:22.71,0:10:24.25,English,,0,0,0,,- It only worked for about a month,
-Dialogue: 0,0:10:22.71,0:10:24.25,Chinese,,0,0,0,,
+Dialogue: 0,0:10:22.71,0:10:24.25,Chinese,,0,0,0,,它只运行了大约一个星期
```

Comment:
*month*

Reply:
sry I will fix this

Reply:
could you edit this line? I can't edit this with your name appended